clients/venice-client/src/main/java/com/linkedin/venice/fastclient/RetriableAvroGenericStoreClient.java

```diff
@@ -83,6 +83,10 @@ public void run() {
   /**
    * TODO:
    * 1. Limit the retry volume.
+   *
+   * QQ based on above TODO: Looks like the retry is being scheduled once via scheduler (LONG_TAIL_RETRY) and
```

Comment:
I suggest to remove all the `QQ` from the javadoc, which could be misleading for other committers.
Also the retry volume limit is meant to reduce the total allowed retry per node.
Today, the retry volume can be 1 retry request per the original request at max, which is unexpected, and we would like to design some mechanism to make it configurable, such as retry at most the slowest 5% of traffic, otherwise, too many retry requests could cause cascading failure.

Reply:
Discussion and questions on the code should be moved to comments in the PR not in the code. Like gaojie said the QQ comments will need to be removed before merge.

Reply:
Makes sense thanks. Yes, these were added as a conversation starter for this PR, will remove this in the next iteration.